actions/views/dashboard.js

```diff
@@ -412,7 +412,7 @@ export const loadMeetingData = (uid, meetingId) => (dispatch) => {
     console.log('finding utterances for meeting', meetingId);
     return app.
         service('utterances').
-        find({query: {meeting: meetingId, $limit: 10000}}).
+        find({query: {meeting: meetingId, $limit: 10000, stitch: true}}).
```

Comment:
I think that would work but `riff-rtc` has 1 additional parameter `$sort`, which may be needed as well (I haven't gone to actually see how the 2 parameters may interact).
```
    return app.service('utterances').find({query:
                                           {meeting: meetingId,
                                            $limit: 10000,
                                            stitch: true,
                                            $sort: { startTime: 1 }}})
```

Reply:
I looked at the code and it doesn't look like sort is required:

https://github.com/rifflearning/riff-server/blob/develop/src/services/utterance/hooks/stitchHook.js#L6

It doesn't check for a sort param and it sorts it regardless

so it's still not clear to me how exactly they interact (if the $sort param is processed before the hook, it would mean the sort specified would never matter), but i believe it should work

Reply:
Part of the reason I ask is that I vaguely remember when Dan first implemented this something wasn't right, and then he said something about sorting the utterances to fix it. So when I saw that $sort property I thought it might be needed. The thing to do is probably to look at the commit history and see when $sort was added.